Tests/Unit/Core/Helper_Test.php

```diff
@@ -43,4 +43,26 @@ public function testGetPluginPayments()
             $this->assertTrue(!!$payment->oxpayments__wdoxidee_iswirecard->value);
         }
     }
+
+    /**
+     * @dataProvider testGetFloatFromStringProvider
+     */
+    public function testGetFloatFromString($input, $expected)
+    {
+        $this->assertEquals(Helper::getFloatFromString($input), $expected);
+    }
+
+    public function testGetFloatFromStringProvider()
+    {
+        // note that these datasets do not return correct values but the ones
+        // calculated by OXIDs "European Format" conversion method
+        return [
+            'thousands English' => ['1,000', 1],
```

Comment:
Please include a test for 3,000.00

Reply:
This is already covered by the `mixed English` set.